tests/common/gcp_api/compute_test.py

```diff
@@ -117,6 +117,21 @@ def test_get_firewall_rules_errors(self, name, response, status,
         with self.assertRaises(expected_exception):
             list(self.gce_api_client.get_firewall_rules(self.project_id))
 
+    def test_get_quota(self):
```

Comment:
Please add one more test where you call get_quota for a metric that does not exist and verify that a KeyError is raised.